docs/game-servers/install-a-half-life-2-deathmatch-dedicated-server-on-debian-or-ubuntu.md

```diff
@@ -38,7 +38,7 @@ This guide will show you how to set up your own [Half-Life 2 Deathmatch](http://
         sudo iptables -R INPUT 5 -p udp -m udp --sport 27000:27030 --dport 1025:65355 -j ACCEPT
         sudo iptables -I INPUT 7 -p udp -m udp --dport 27000:27030 -j ACCEPT
 
-## Install the Server
+## Install the Steam Server
```

Comment:
It's not a Steam Server.

Reply:
It's an Half-Life 2: Deathmatch server installed from Steam's command line client (SteamCMD).